cfpb/cfgov-refresh
# Adding third width link blob group

Adding third width link blob group, which is necessary for the http://www.consumerfinance.gov/about-us/careers/working-at-cfpb/ page.

## Additions

- Added `cfgov/jinja2/v1/_includes/organisms/third-width-link-blob-group.html` to handle three column layouts.
- Added `cfgov/jinja2/v1/_includes/organisms/info-unit-group.html` to serve as a generic blog group builder.

## Changes

- Modified `cfgov/v1/atomic_elements/molecules.py` to use the `link-blob` template.
- Modified `cfgov/unprocessed/css/organisms/info-unit-group.less` to reflect the add three column layout. 
- Changed the name of `cfgov/_includes/molecules/half-link-blob.html` to  `half-width-link-blob-group.html`
- Modified `cfgov/jinja2/v1/_includes/organisms/half-width-link-blob-group.html` to call `info-unit-group.html` macro.

## Testing

~~~Currently unable to test the third width link blob group. Requires changes in Wagtail.~~~ Instructions for the third-width link blob group are below:

- Create a `Browse` page via Wagtail. 
- Add a `Third Width Link Blob Group` molecule. 
- Add 3 half-width link blobs. Preview.
- Notice that the organism renders three link blobs in separate columns.
- Return to Wagtail
- Check the `Has top border` and `Has bottom border` checkboxes, preview again.
- Notice that the organism renders as before but with borders.

## Review

- @chosak 
- @richaagarwal 
- @virginiacc 
- @jimmynotjim 


## Checklist
